bn-models/src/models/_lookups/vessel.ts

```diff
@@ -35,12 +34,12 @@ export interface Vessel extends BaseLookup {
   stateRegulationNumber?: string;
   registeredLength?: Measurement;
   vesselStatus?: string; // lookup value - Active, Inactive, Retired, Sank, Byback
-  captains?: VesselCaptain[];
   safetyMeasures?: SafetyMeasure[]; // 30+ items
   media?: Media[];
   isActive?: boolean;
   emHardware?: string;
   thirdPartyReviewer?: string;
+  passCode?: string // user must supply correct vessel passcode to gain access to vessel.
```

Comment:
Interesting.

Reply:
Yeah, this is what OLE does currently in their over the phone system. Not sure why we didn't think to apply it to our system sooner... I think it makes a lot of sense.